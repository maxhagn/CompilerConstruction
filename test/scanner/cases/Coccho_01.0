end35
beginning_7237ssd_sfdsw43
so_______many___________________underscores____
_variables_can_start_with_underscores
_______________THEY_CAN_START_WITH_MANY_UNDERSCORES
________34234234sdsdsadasdds
_2313____234fdfd