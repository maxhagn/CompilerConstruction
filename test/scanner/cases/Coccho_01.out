id end35
id beginning_7237ssd_sfdsw43
id so_______many___________________underscores____
id _variables_can_start_with_underscores
id _______________THEY_CAN_START_WITH_MANY_UNDERSCORES
id ________34234234sdsdsadasdds
id _2313____234fdfd